### PR TITLE
Les 6

### PR DESCRIPTION
**Pull request by Ihorzolotar**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/ZolotarModule/AskQuestion/view/frontend/web/js/validate-ukraine-number.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+define([
+    'jquery',
+    'jquery/validate'
+], function ($) {
+    'use strict';
+    console.log('Script validate ukraine number is run');
```
Comment (maksymz) [inline app/code/ZolotarModule/AskQuestion/view/frontend/web/js/validate-ukraine-number.js]: Missed functionality.

File: app/code/ZolotarModule/AskQuestion/view/frontend/web/js/askQuestionScript.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+             */
+            submitForm: function () {
+                var reg = /^\+380\d{3}\d{2}\d{2}\d{2}$/;
+                var phone = document.forms.ask_question.telephone.value;
+                var CurrentTime = new Date();
+                var cookieSent = $.mage.cookies.get(this.options.expires);
+                var formData = new FormData($(this.element).get(0));
+                formData.append('form_key', $.mage.cookies.get('form_key'));
+                formData.append('isAjax', 1);
+                if (!reg.test(phone)) {
```
Comment (maksymz) [inline app/code/ZolotarModule/AskQuestion/view/frontend/web/js/askQuestionScript.js]: This code MUST be a working JS mixin. Such validations are usually used multiple times in the application, so such new validation must not be hardcoded here. See the video that was included in presentation materials - https://www.youtube.com/watch?v=0fguDL5iEd0

Comment (Ihorzolotar) [reply app/code/ZolotarModule/AskQuestion/view/frontend/web/js/askQuestionScript.js]: done

**Comment by Ihorzolotar** [comment]
Please tell me what needs to be fixed in this work.

**Pull request merged**